.github/workflows/deploy.yml

```diff
@@ -0,0 +1,51 @@
+# Simple workflow for deploying static content to GitHub Pages
```

Comment:
@LanesGood Is this Github action supposed to be part of this PR?

Reply:
@oliverroick no - don't know why this was checked in. I think I was basing off another branch, maybe it accidentaly grabbed it from somewhere else?